moped-database/config/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.*
+!.gitignore
+!primary_key_map.json
+!hasura.local.yaml
+!hasura_cluster_template.json
+!README.md
```

Comment:
This .gitignore makes it really hard (TM) to commit a file in this directory. This is due to a previous version of the README.md saying that this directory is for local configuration files and to be careful about putting things in here in git. This was sage advice, however, we are storing some configuration templates in here, so I have amended the README with more information but continue to honor the old intent. I don't want to cause people to accidentally start committing secrets.